packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -74,6 +75,7 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
        assert(currentIndex != null),
        assert(0 <= currentIndex && currentIndex < items.length),
        assert(iconSize != null),
+       assert(height != null && height >= 0.0),
```

Comment:
Should there be any handling for how this relates to the `iconSize`? If the iconSize is greater than the height, what happens?

Reply:
Good point!

The code before this PR does not check the iconSize. That is, it allows the iconSize to be greater than the default TabBar height.

I think it's better not to check to avoid breaking change.

Reply:
I don't think it would be a breaking change, it only would be checking the iconSize in relation to the height, which does not currently exist.
I've checked the equivalent in the material library (height related PR link above), and it does not check the icon size, so I think we can leave it for consistency.